src/gluonts/zebras/_period.py

```diff
@@ -49,7 +49,7 @@
 from typing import Any, Union, cast, overload
 
 import numpy as np
-from dateutil.parser import parse as du_parse
+from dateutil.parser import parse as du_parse  # type: ignore
```

Comment:
What is this one about?

Reply:
I see, but then let's install the stubs instead of doing this `type: ignore`.

Also, it doesn't currently trigger because of `--ignore-missing-imports` I think.